common/djangoapps/student/auth.py

```diff
@@ -89,19 +109,48 @@ def get_user_permissions(user, course_key, org=None):
         return STUDIO_NO_PERMISSIONS
     all_perms = STUDIO_EDIT_ROLES | STUDIO_VIEW_USERS | STUDIO_EDIT_CONTENT | STUDIO_VIEW_CONTENT
     # global staff, org instructors, and course instructors have all permissions:
-    if GlobalStaff().has_user(user) or OrgInstructorRole(org=org).has_user(user):
+    # TODO: course roles: If the course roles feature flag is disabled the organization_permissions_list_check call
+    #       below will never return true.
+    #       Remove the OrgInstructorRole .has_user call when course roles are implemented.
+    if (
+        GlobalStaff().has_user(user)
+        or OrgInstructorRole(org=org).has_user(user)
+        or organization_permissions_list_check(user, COURSE_INSTRUCTOR_ROLE_PERMISSIONS, org)
+    ):
         return all_perms
-    if course_key and user_has_role(user, CourseInstructorRole(course_key)):
+
+    # TODO: course roles: If the course roles feature flag is disabled the course_permissions_list_check call
+    #       below will never return true.
+    #       Remove the user_has_role call when course roles are implemented.
+    if course_key and (
+        user_has_role(user, CourseInstructorRole(course_key))
+        or course_permissions_list_check(user, COURSE_INSTRUCTOR_ROLE_PERMISSIONS, course_key)
+    ):
         return all_perms
+
     # Limited Course Staff does not have access to Studio.
+    # TODO: course roles: Remove this validation when course roles are implemented
```

Comment:
Consider adding clarity by explaining which type of course roles you mean (student_courseaccessrole, django_comment_client_role, or course_roles_roles)

Reply:
I wanted to refer to the course_roles Django App . What you think about changing the comment to:  "# TODO: course roles: Remove this validation when implementing course_roles django app."?

Reply:
I think that is a positive change.

Reply:
done